polling_stations/templates/fragments/polling_station_known.html

```diff
@@ -8,54 +8,98 @@ <h2>
     {% endblocktrans %}
 </h2>
 
-{% if not show_polls_open_card and advance_voting_station %}
-    <h3>{% trans "Vote in advance" %}</h3>
+{% if not show_polls_open_card and advance_voting_station and advance_voting_station.open_in_future %}
+    <p>
+        {% if NEXT_CHARISMATIC_ELECTION_DATE %}
+            {% blocktrans trimmed %}
+                Your council is trialling a system that allows you to vote in person before polling day.
+                You can <a href="#vote-in-advance">vote in advance</a>
+                at this location, or <a href="#vote-on-polling-day">vote at your polling station</a>
+                as normal on&nbsp;{{ NEXT_CHARISMATIC_ELECTION_DATE }}.
+            {% endblocktrans %}
+        {% else %}
+            {% blocktrans trimmed %}
+                Your council is trialling a system that allows you to vote in person before polling day.
+                You can <a href="#vote-in-advance">vote in advance</a>
+                at this location, or <a href="#vote-on-polling-day">vote at your polling station</a>
+                as normal on&nbsp;polling day.
+            {% endblocktrans %}
+        {% endif %}
+    </p>
     {% include "fragments/advance_voting_station.html" %}
```

Comment:
This fragment is only included in `polling_station_known.html`. However it's possible we will have assigned an advance station to an address, but not a polling station - e.g. when every address in a council can vote at a single advance station. However I think this could end up being really confusing to users, so I'm not sure what's best - but realised I hadn't previously considered it - maybe you had?

Reply:
My feeling is that we should keep it simple: if we don't know the polling station, we don't know about advance stations. Not ideal, I know, but I think it's the only way to keep it simple.

We also don't show the advance station in the API if the station isn't known, I think

Reply:
We don't show the advance station on a postcode view, if the postcode is split between polling station known and unknown (even if all addresses go to the same advance station). However if you go to the `address` endpoint for a uprn where we don't know the polling station, then you will get an advance station if we know it. This is also true when none of the addresses in the postcode have a known polling station - however this is less plausible, because we don't return a list of uprns.

Reply:
However I agree that we shouldn't show it on the front end.